<launch>
<node pkg="hector_costmap" type="hector_costmap" name="hector_costmap" output="screen" respawn="false">
    <param name="initial_free_cells_radius" value="0.3" />
    <param name="update_radius" value="4.0"/>
    <param name="max_delta_elevation" value="0.07"/>
    <param name="allow_kinect_to_clear_occupied_cells" value="true" />
    <param name="max_clear_size" value="4" />
    <param name="use_grid_map" value="true" />
    <param name="use_elevation_map" value="true" />
    <param name="cost_map_topic" value="cost_map" />
    <param name="elevation_map_topic" value="elevation_map_local" />
    <param name="grid_map_topic" value="scanmatcher_map" />
    <param name="map_frame_id" value="map" />
    <param name="local_map_frame_id" value="base_footprint" />
    <param name="sysMsgTopic" value="syscommand" />
    <param name="costmap_pub_freq" value="4.0" />
<!-- Not necessary to set, set by scanmatcher -->
    <!--param name="map_resolution" value="0.05" /-->
    <!--param name="max_grid_size_x" value="1024" /-->
    <!--param name="max_grid_size_y" value="1024" /-->
<!-- Not necessary to set, set by elvation_mapping -->
    <!-- param name="elevation_resolution" value="0.01" / -->
</node>
</launch>
